src/libraries/Native/Unix/System.Native/pal_memory.h

```diff
@@ -7,21 +7,36 @@
 #include "pal_types.h"
 
 /**
- * C runtime malloc
+ * C runtime aligned_alloc
  */
-PALEXPORT void* SystemNative_MemAlloc(uintptr_t size);
+PALEXPORT void* SystemNative_AlignedAlloc(uintptr_t alignment, uintptr_t size);
 
 /**
- * C runtime realloc
+ * C runtime free for aligned_alloc
  */
-PALEXPORT void* SystemNative_MemReAlloc(void* ptr, uintptr_t size);
+PALEXPORT void SystemNative_AlignedFree(void* ptr);
+
+/**
+ * C runtime calloc
+ */
+PALEXPORT void* SystemNative_Calloc(uintptr_t num, uintptr_t size);
 
 /**
  * C runtime free
  */
-PALEXPORT void SystemNative_MemFree(void* ptr);
+PALEXPORT void SystemNative_Free(void* ptr);
 
 /**
  * C runtime memset
  */
 PALEXPORT void* SystemNative_MemSet(void* s, int c, uintptr_t n);
+
+/**
+ * C runtime malloc
+ */
+PALEXPORT void* SystemNative_Malloc(uintptr_t size);
```

Comment:
Nit: Malloc is before MemSet in the .c file

Reply:
Fixed.